charts/agent-control/templates/_helpers.tpl

```diff
@@ -16,6 +16,10 @@ overrides:
 
   {{- if not .Values.agentControlCd.enabled -}}
     {{- $_ := set $config "cdRemoteUpdate" false -}}
+  {{- else -}}
```

Comment:
Work on this comment from @paologallinaharbur https://github.com/newrelic/helm-charts/pull/1886#discussion_r2298476892